recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "multiscale-spatial-image" %}
-{% set version = "1.0.0" %}
+{% set version = "2.0.2" %}
```

Comment:
This PR bumped the version without updating the sha. It only appeared to have passing CI because of the `[ci skip]` added to the most recent commits. No new binaries were uploaded

Reply:
ouch, thanks for letting me know!